components/brave_ads/browser/ad_notification.cc

```diff
@@ -45,7 +45,7 @@ std::unique_ptr<message_center::Notification> CreateAdNotification(
                                  kNotifierId),
       notification_data, nullptr);
 
-#if !defined(OS_MACOSX) || defined(OFFICIAL_BUILD)
+#if !defined(OS_APPLE) || defined(OFFICIAL_BUILD)
```

Comment:
Should be `OS_MAC`

Reply:
Address by 0230f8e014a698a5cf5abd780734aa5bae433b3f